test/http-server.test.ts

```diff
@@ -29,6 +30,27 @@ describe('HTTP Server', () => {
     expect(server.listening).toBe(false);
   });
 
+  it("should support https if cert + key options are supplied", async () => {
```

Comment:
Could you please switch to single quotes wherever they are used? Just to be consistent with the rest of the code.

Reply:
Fixed!